src/GitHub.App/ViewModels/GitHubPane/NoRemoteOriginViewModel.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Reactive;
+using System.Threading.Tasks;
+using System.ComponentModel.Composition;
+using GitHub.Services;
+using ReactiveUI;
+
+namespace GitHub.ViewModels.GitHubPane
+{
+    /// <summary>
+    /// The view model for the "No Origin Remote" view in the GitHub pane.
+    /// </summary>
+    [Export(typeof(INoRemoteOriginViewModel))]
+    [PartCreationPolicy(CreationPolicy.NonShared)]
+    public class NoRemoteOriginViewModel : PanePageViewModelBase, INoRemoteOriginViewModel
+    {
+        ITeamExplorerServices teamExplorerServices;
+
+        [ImportingConstructor]
+        public NoRemoteOriginViewModel(ITeamExplorerServices teamExplorerServices)
+        {
+            this.teamExplorerServices = teamExplorerServices;
+            EditRemotes = ReactiveCommand.CreateFromTask(OnEditRemotesAsync);
```

Comment:
Just curious, why not `ReactiveCommand.CreateFromTask(teamExplorerServices.ShowRepositorySettingsRemotesAsync)`?